NEWS.md

```diff
@@ -12,9 +12,10 @@ for human readability.
 
 - Added the three-dimensional multi-ion magneto-hydrodynamics (MHD) equations with a
   generalized Lagrange multipliers (GLM) divergence cleaning technique ([#2215]).
-- New time integrator `PairedExplicitRK4`, implementing the fourth-order 
+- New time integrator `PairedExplicitRK4`, implementing the fourth-order
   paired explicit Runge-Kutta method with [Convex.jl](https://github.com/jump-dev/Convex.jl)
   and [ECOS.jl](https://github.com/jump-dev/ECOS.jl) ([#2147])
+- Passive tracers for arbitrary equations with density and flow variables
```

Comment:
```suggestion
- Passive tracers for arbitrary equations with density and flow variables ([#2364])
```
We usually put the PR number behind the description. These are automatically converted to links in the docs.